test/library/helpers.go

```
@@ -0,0 +1,323 @@
+	}
+	return strings.TrimSpace(token)
+}
+
+func RemoteShPod(oc *OC, namespace, pod, shell string, args ...string) (string, error) {
+	execArgs := append([]string{"-n", namespace, "pod/" + pod, "--", shell}, args...)
+	return oc.AsAdmin().WithoutNamespace().Run("exec").Args(execArgs...).Output()
+}
+
+func CheckMetric(oc *OC, url, token, metricString string, timeoutSec int) {
+	getCmd := "curl -G -k -s -H \"Authorization:Bearer " + token + "\" " + url
+	err := wait.Poll(10*time.Second, time.Duration(timeoutSec)*time.Second, func() (bool, error) {
```

golangci-lint (2.12.2)

[error] SA1019: wait.Poll is deprecated: This method does not return errors from context, use PollUntilContextTimeout. Note that the new method will no longer return ErrWaitTimeout and instead return errors defined by the context package. Will be removed in a future release. 

(staticcheck)